### PR TITLE
Model lists in Python tests

### PR DESCRIPTION
**Pull request by popovaan**
Extended tests with model lists.

Ticket: CVS-138675

### CODE REVIEW

File: text_generation/causal_lm/cpp/continuous_batching/python/tests/test_sampling.py
Comment (ilya-lavrenov) [inline text_generation/causal_lm/cpp/continuous_batching/python/tests/test_sampling.py:1]: maybe we can have 3 test files:
- lists of models
- preemption
- sampling

File: text_generation/causal_lm/cpp/continuous_batching/python/tests/models/nightly [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+hf-internal-testing/tiny-random-PLBartForCausalLM
+hf-internal-testing/tiny-random-SwinBackbone
+hf-internal-testing/tiny-random-electra
+hf-internal-testing/tiny-random-FNetForMaskedLM
+hf-internal-testing/tiny-random-PersimmonForSequenceClassification
+hf-internal-testing/tiny-random-MobileViTV2ForImageClassification
+hf-internal-testing/tiny-random-XGLMForCausalLM
+hf-internal-testing/tiny-random-SEWForCTC
+hf-internal-testing/tiny-random-UMT5ForSequenceClassification
+hf-internal-testing/tiny-random-PersimmonModel
```
Comment (ilya-lavrenov) [inline text_generation/causal_lm/cpp/continuous_batching/python/tests/models/nightly]: can we keep only CausalLM models https://huggingface.co/hf-internal-testing?search_models=CausalLM as agreed before ?

File: text_generation/causal_lm/cpp/continuous_batching/python/python.cpp [hunk 2]
```diff
@@ -26,6 +26,18 @@ PYBIND11_MODULE(py_continuous_batching, m) {
         .def(py::init<>())
         .def_readonly("m_request_id", &GenerationResult::m_request_id)
         .def_readwrite("m_generation_ids", &GenerationResult::m_generation_ids)
+
+        .def("get_generation_ids", 
+            [](GenerationResult &r) -> py::list {
+                py::list res;
+                for (auto s: r.m_generation_ids) {
+                
+                    PyObject* py_s = PyUnicode_DecodeUTF8(s.data(), s.length(), "replace");
```
Comment (ilya-lavrenov) [inline text_generation/causal_lm/cpp/continuous_batching/python/python.cpp:33]: is it possible to fix `__repr__` as well?

File: text_generation/causal_lm/cpp/continuous_batching/python/tests/test_sampling.py
Comment (popovaan) [reply text_generation/causal_lm/cpp/continuous_batching/python/tests/test_sampling.py:1]: Moved tests to separate files "test_preemtion.py", "test_sampling.py". Model lists are in a separate folder.

File: text_generation/causal_lm/cpp/continuous_batching/python/tests/models/nightly (hunk 1 above)
Comment (popovaan) [reply text_generation/causal_lm/cpp/continuous_batching/python/tests/models/nightly]: Yes, done.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**